locale/en/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,1311 @@
+# Berth Reservations API
+# Copyright (C) 2020 City of Helsinki
+# This file is distributed under the same license as the berth-reservations package.
```

Comment:
why do we need a file for `en`? 🤔

Reply:
My intuition tells me because `fi` is the default language 🤔  but not acutally sure, I just created the file for the three languages we support.

Reply:
When you do `_("My string")` you tell Django that this is a translatable string with id `My string`. Then Django checks for active language and checks its .mo/.po files. If no file is given it will use this id as string itself. Since we agreed beforehand that all our ids are in english, we can basically skip locale files for `en`.

Reply:
Ahaa! Okay, okay, makes sense then 👍